Dockerfiles/Dockerfile

```diff
@@ -1,9 +1,24 @@
+FROM golang:1.15-alpine as myiac-builder
+
+# Placeholders to include extra folders in final container (i.e. charts)
+ARG EXTRA_WORKDIR_ORIG=/workdir/Dockerfiles
+ARG EXTRA_WORKDIR_DEST=/home/app/Dockerfiles
```

Comment:
@pozgo something very weird happens with these parameters (the default values are placeholders), when passing them and going into the non-builder `FROM`, looks like the contents of `/home/app/charts` (the real value passed in by script `build-myiac-moneycol-docker.sh`) is overridden. I tested this by:

- Running the mentioned script
- Commenting out all the 'second' part of the `Dockerfile`. I can check `ls /home/app` and see `charts` there. 
- If I leave the build as-is, no longer see `/home/app/charts`. Looks like something from the `apk` onwards overrides the contents (`/home/app/account.json` is still there...)

I am a bit puzzled, could use another pair of eyes in case you spot something:
- Just checkout the branch
- run `./build-myiac-moneycol-docker.sh`
- check the content of the folder `/home/app` and see that there's no `charts`

Reply:
I realised I messed up the `charts` subfolder with submodules somehow... urg... this should've been the easy part! Anyway, if you think that something is wrong in that Dockerfile, please share :)

Reply:
LGTM